src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const notification = document.createElement('div');
+
+  body.appendChild(notification);
+
+  notification.classList.add(`notification`, type);
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const notificationTitle = document.createElement('h2');
+
+  notificationTitle.className = 'title';
+  notificationTitle.innerText = title;
+
+  const notificationDesc = document.createElement('p');
+
+  notificationDesc.innerText = description;
+
+  notification.appendChild(notificationTitle);
+  notification.appendChild(notificationDesc);
```

Comment:
You shoud append notification after adding title and description
```suggestion

  notification.classList.add(`notification`, type);
  notification.style.top = `${posTop}px`;
  notification.style.right = `${posRight}px`;

  const notificationTitle = document.createElement('h2');

  notificationTitle.className = 'title';
  notificationTitle.innerText = title;

  const notificationDesc = document.createElement('p');

  notificationDesc.innerText = description;

  notification.appendChild(notificationTitle);
  notification.appendChild(notificationDesc);
  
  body.appendChild(notification);
```